crates/swc_ecma_codegen/src/stmt.rs

```diff
@@ -42,13 +42,11 @@ mod tests {
     fn declaration_statement() {
         assert_min("var foo;", "var foo");
         assert_min("let foo;", "let foo");
-        assert_min("const foo;", "const foo");
```

Comment:
❓ I removed these tests because they test against parsing module with strict mode as default which means these are invalid syntax now. I can change this if you want them to still be included?

Reply:
It should be removed, thanks!